app-editors/vscode/files/code-url-handler.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Visual Studio Code - URL Handler
+Comment=Code Editing. Redefined.
+GenericName=Text Editor
+Exec=/usr/bin/code --no-sandbox --open-url %U
```

Comment:
Noted for the `vscode` symlink. I have knowledge about the `--no-sandbox` option from Discord: the application lags behind user input when interacting with the app. Since both Discord and vscode are electron based, I suppose the reason is the same. I have not tried to run it without that argument. I will try and see, then.

Reply:
Okay it seems that even if we remove the `--no-sandbox` argument. It starts with the `--no-sandbox` argument anyway ? Except if I messed something up, can you check ?

Reply:
It seems to be related to this: https://github.com/microsoft/vscode/pull/81096
I haven't checked if this PR has been reverted since, but it does not seem like it.